src/main/java/seedu/trackermon/model/show/RatingContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,32 @@
+package seedu.trackermon.model.show;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.trackermon.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code Show}'s {@code Name} matches any of the keywords given.
```

Comment:
Just clarifying shouldn't this be Rating rather than Name?

Reply:
Yes, good spot!